orca/orca-clouddriver/src/test/java/com/netflix/spinnaker/orca/clouddriver/CloudDriverCacheServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package com.netflix.spinnaker.orca.clouddriver;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.urlMatching;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.catchThrowable;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
+import com.github.tomakehurst.wiremock.junit5.WireMockExtension;
+import com.netflix.spinnaker.kork.retrofit.ErrorHandlingExecutorCallAdapterFactory;
+import com.netflix.spinnaker.kork.retrofit.Retrofit2SyncCall;
+import java.util.List;
+import java.util.Map;
+import okhttp3.OkHttpClient;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import retrofit2.Retrofit;
+import retrofit2.converter.jackson.JacksonConverterFactory;
+
+public class CloudDriverCacheServiceTest {
+  @RegisterExtension
+  static WireMockExtension wmCache =
+      WireMockExtension.newInstance().options(new WireMockConfiguration().dynamicPort()).build();
+
+  private static CloudDriverCacheService cacheService;
+
+  @BeforeAll
+  public static void setup() {
+    cacheService =
+        new Retrofit.Builder()
+            .baseUrl(wmCache.baseUrl())
+            .client(new OkHttpClient())
+            .addConverterFactory(JacksonConverterFactory.create(new ObjectMapper()))
+            .addCallAdapterFactory(ErrorHandlingExecutorCallAdapterFactory.getInstance())
+            .build()
+            .create(CloudDriverCacheService.class);
+  }
+
+  @Test
+  public void verifyForceCacheUpdate() {
+    wmCache.stubFor(
+        WireMock.post(urlMatching("/cache/aws/CloudFormation"))
+            .willReturn(WireMock.aResponse().withStatus(200).withBody("{\"status\":\"ok\"}")));
+
+    // FIXME: Retrofit does not support wildcard in the method parameter.
+    Throwable thrown =
+        catchThrowable(
+            () ->
+                Retrofit2SyncCall.execute(
+                    cacheService.forceCacheUpdate(
+                        "aws",
+                        "CloudFormation",
+                        Map.of("id", "1", "region", List.of("us-east-1")))));
+    assertThat(thrown).isInstanceOf(IllegalArgumentException.class);
```

Comment:
Mind adding a verify to show that the wiremock endpoint really does get hit?

Reply:
Added verify statement post fix

Reply:
wiremock does support `verify(exactly(0), ...)`, but verifying post-fix is fine.